packages/core/admin/admin/src/pages/Settings/components/SettingsNav.tsx

```diff
@@ -67,7 +79,19 @@ const SettingsNav = ({ menu }: SettingsNavProps) => {
                   endAction={
                     <>
                       {link?.licenseOnly && (
-                        <Lightning fill="primary600" width="1.5rem" height="1.5rem" />
+                        <Lightning
+                          fill={
+                            (availableFeatureNames || []).includes(
+                              linksIdsToLicenseFeaturesNames[
+                                link.id as keyof typeof linksIdsToLicenseFeaturesNames
```

Comment:
just in terms of readability can you create a type definition for this type on top?